docs/docs/modelling-the-domain.md

```diff
@@ -250,6 +250,8 @@ class AddAuthor < Sequent::Command
 end
 ```
 
+And the `AuthorCommandHandler` to `config/initializers/sequent.rb`.
```

Comment:
This is a mandatory step and was missing, as reported via https://github.com/zilverline/sequent/issues/212.